test/integration/material-ui/components.spec.tsx

```diff
@@ -322,21 +323,21 @@ function DialogTest() {
       <div>
         <List>
           {emails.map((email) => (
-            <ListItem button onClick={(e) => log(e)} key={email}>
+            <ListItemButton onClick={(e) => log(e)} key={email}>
               <ListItemAvatar>
                 <Avatar>
                   <FakeIcon />
                 </Avatar>
               </ListItemAvatar>
               <ListItemText primary={email} />
-            </ListItem>
+            </ListItemButton>
           ))}
           <ListItem
             ref={(elem) => {
-              expectType<HTMLLIElement | null, typeof elem>(elem);
+              expectType<HTMLDivElement | null, typeof elem>(elem);
             }}
             onClick={(e) => {
-              expectType<React.MouseEvent<HTMLLIElement, MouseEvent>, typeof e>(e);
+              expectType<React.MouseEvent<HTMLDivElement, MouseEvent>, typeof e>(e);
```

Comment:
Shouldn't these be `HTMLLIElement` still?

Reply:
Yes, my bad. Because I didn't remove the `ExtendButtonBase<ListItemTypeMap<{}, 'div'>> &`, `e` and `elem` was of type `HTMLDivElement`. Removing that earlier line fixes this, so I have reverted it back to `HTMLLIElement`.